poly-commitment/src/commitment.rs

```diff
@@ -241,7 +239,7 @@ impl<'a, 'b, C: AffineCurve> Sub<&'a PolyComm<C>> for &'b PolyComm<C> {
         let n2 = other.elems.len();
         for i in 0..std::cmp::max(n1, n2) {
             let pt = if i < n1 && i < n2 {
-                self.elems[i] + (-other.elems[i])
+                (self.elems[i] - other.elems[i]).into_affine()
```

Comment:
Is it not less efficient? IIUC, it goes into projective coordinates and come back into affine, performing an inverse for each element?

Reply:
See